box-share-sdk/src/main/java/com/box/androidsdk/share/vm/DataWrapper.java

```diff
@@ -0,0 +1,55 @@
+package com.box.androidsdk.share.vm;
+
+import androidx.annotation.StringRes;
+
+/**
+ * A data wrapper to make sure LiveData changes can take network responses into account
```

Comment:
If this is more generic than network errors , I'd change the comment

Reply:
Got it. I will change the comment.